### ISSUE TITLE
Evans syndrome

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6076]
**Preferred term label:**
Evans syndrome

**Synonyms**
 Autoimmune hemolytic anemia, Autoimmune thrombocytopenia, Immune pancytopenia

**Definition (free text, please give PubMed ID)**
A rare and chronic autoimmune disease characterized by autoimmune hemolytic anemia and autoimmune thrombocytopenic purpura with a positive direct anti-human globulin test
PMID: 30349415 

**Parent term (use hpo.jax.org/app)**
Autoimmunity

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 601608, ORPHA: 1959

**Comment by pnrobinson** [comment]
This is a disease not a phenotypic feature. See the MONDO term for the disease:
https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0016030
